README.md

```diff
@@ -12,6 +12,10 @@ The template repository includes:
   * [main.tf](main.tf): the primary entry point for the module
   * [variables.tf](variables.tf): contains variable declarations, including the required `var.services` for Consul Terraform Sync compatibility.
 
+This repository has different branches that can be checked out to see templates for various features that may have different supported variable and documentation recommendations.
+ * `main` branch: basic template for default services condition and intermediate input variables
+ * `catalog-services-condition-template` branch: template for catalog-service condition
```

Comment:
Linking to the branch might help with discovery!
```suggestion
 * [`catalog-services-condition-template`](https://github.com/hashicorp/consul-terraform-sync-template-module/tree/catalog-services-condition-template) branch: template for catalog-service condition
```

Reply:
Good idea! Thank you